packages/test-runner-intercept/browser/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Intercept a module in order to change its implementation.
+ * @param {string} moduleName The name of the module to intercept. When intercepting the import of a module, the name should exactly match the name of the import.
+ * @returns {Promise<Record<string, func> | void >} Writable object with every named export of the intercepted module as a property.
+ * If the module to be intercepted contains a default export, the export will be available in the `default` property. Only function expressions are supported when intercepting default exports.
+ */
+export async function interceptModule(moduleName) {
+  let module;
+  try {
+    module = await import(`/__intercept__/${moduleName}`);
```

Comment:
```suggestion
    module = await import(`/__intercept-module__/${moduleName}`);
```

Reply:
Yup, that's good. I'll push a change that modifies all three instances.

Reply:
done! :-)

Reply:
thanks!